Frontend/ui-driver/src/Flow.purs

```diff
@@ -1744,8 +1776,8 @@ onBoardingSubscriptionScreenFlow onBoardingSubscriptionViewCount = do
         Nothing -> onBoardingSubscriptionScreenFlow (state.props.screenCount-1)
   pure unit
 
-homeScreenFlow :: FlowBT String Unit
-homeScreenFlow = do
+homeScreenFlow :: Boolean -> FlowBT String Unit
+homeScreenFlow check = do
```

Comment:
why we need to pass boolean value to check wheather to call event or not ?

Reply:
instead just pass this boolean to js function which will update the key and make flag true, so next when we check and if the flag is true will call the event

Reply:
We need to call this after all the API calls are done

Reply:
maintain seperate flag for this